lib/prune.js

```diff
@@ -21,6 +21,8 @@ prune.completion = require('./utils/completion/installed-deep.js')
 
 function prune (args, cb) {
   var dryrun = !!npm.config.get('dry-run')
+  // prune should not touch the lock file
+  npm.config.set('save', false)
```

Comment:
Btw, isn't setting this config option a permanent thing? Or is it just for this command and this run?